File: Assignment 2/Switch case/area.cpp
#include<iostream>
using namespace std;
class Area
{
    int choice;
    float num1,num2,area;
    public:void accept()
    {
        cout<<"Choose a polygon to determine its area:\n"
        "1. Square\n"
        "2. Rectangle\n"
        "3. Triangle\n"
        "4. Exit"
        "\n\nChoice=";
        cin>>choice;

        while(choice<1||choice>4)
        {
        cout<<"Please enter a valid choice"
            "\n\nChoice=";
        cin>>choice;
        }

        switch(choice)
        {
        case 1:
            cout<<"Enter the length of any two sides of a Square:";
            cin>>num1>>num2;
            area=num1*num2;
            cout<<"The area of Square is: "<<area;
            break;

        case 2:
            cout<<"Enter the length and the width of a Rectangle:";
            cin>>num1>>num2;
            area=num1*num2;
            cout<<"The area of Rectangle is: "<<area;
            break;

        case 3:

            cout<<"Enter the length of base of a Triangle:";
            cin>>num1;
            cout<<"Enter the height of a triangle:";
            cin>>num2;
            area=(0.5*num1*num2);
            cout<<"The area of Triangle is:"<<area;
            break;

        default:cout<<"error";


        }


    }
};
int main()
{
    Area obj;
    obj.accept();
}

File: Java assignment(1-2)/java Assignment 2/Area.java
import java.util.Scanner;
public class Area
{
float num1,num2,areaoftria,areaofrect,areaofsqua;
int option;
void getData()
{
Scanner sc=new Scanner(System.in);
System.out.println("---------Menu---------");
System.out.println("1. Area of a triangle");
System.out.println("2. Area of a Square ");
System.out.println("3. Area of a Rectangle");
System.out.println("Choose an option: ");
option=sc.nextInt();
switch(option)
{
case 1: tria();
break;
case 2: squa();
break;
case 3: rect();
break;
default: System.out.println("Enter a vaild option");
getData();
}
}
void tria()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter the base");
num1=sc.nextFloat();
System.out.println("Enter the height");
num2=sc.nextFloat();
areaoftria=0.5f*num1*num2;
System.out.println("Area of triangle is "+areaoftria);
}
void squa()
{ 
Scanner sc=new Scanner(System.in);
System.out.println("Enter the length of one side: ");
num1=sc.nextFloat();
System.out.println("Enter the length of the other side: ");
num2=sc.nextFloat();
areaofsqua=num1*num2;
System.out.println("Area of Square is "+areaofsqua);
}
void rect()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter the length: ");
num1=sc.nextFloat();
System.out.println("Enter the breadth: ");
num2=sc.nextFloat();
areaofrect=num1*num2;
System.out.println("Area of Rectangle is "+areaofrect);
}

public static void main(String...args)
{
Area obj=new Area();
obj.getData();
}
}












